src/web/theme/modules/ProductSynthesis/ProductInfo/ProductInfo.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { H2 } from "theme/ui/atoms/Typography/Heading";
+import Actions from "../Actions";
+import Price from "theme/ui/atoms/Typography/Price";
+import infos from "./ProductInfo.json";
```

Comment:
Shouldn't this data come from the back-end ? this may have been discussed with others, if so disregard my comment :smile: